types/k6/http.d.ts

```diff
@@ -39,7 +39,7 @@ export function get<RT extends ResponseType | undefined>(
  */
 export function options<RT extends ResponseType | undefined>(
     url: string,
-    body?: RequestBody | null,
+    body?: RequestBody | null | ArrayBuffer,
```

Comment:
I am not sure, but instead of adding `ArrayBuffer` here and in a bunch of places below, wouldn't simply adding `ArrayBuffer` to this type be enough: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/79a43b97809c7c9dce831502329e6848e4edd2dd/types/k6/http.d.ts#L275-L278

Reply:
You are right. I started skipping the `del` method because https://k6.io/docs/javascript-api/k6-http/del-url-body-params but I guess it does not matter.

Reply:
fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51236/commits/92cc97dfdf0eff508be6c21f45fdeafe4a57f7c7